docker-compose.yml

```diff
@@ -8,7 +8,7 @@ x-logging: &default-logging
 
 services:
   loket:
-    image: lblod/frontend-loket:0.92.1
+    image: lblod/frontend-loket:feature-impersonation-refactor
```

Comment:
I used a feature image for easier testing, we should replace it with a tagged release before merging this.